torch/csrc/distributed/c10d/NCCLUtils.hpp

```diff
@@ -366,6 +398,11 @@ class NCCLComm {
     return aborted_;
   }
 
+  bool isInitialized() const {
```

Comment:
is this used in another PR or is it extraneous?

Reply:
removed